docs/DeveloperGuide.md

```diff
@@ -316,7 +318,31 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 * 3a. The given index is invalid.
 
-    * 3a1. AddressBook shows an error message.
+    * 3a1. FAKEJARVIS shows an error message.
+
+      Use case resumes at step 2.
+
+
+**Use case: Delete a task**
+
+**MSS**
+
+1.  User requests to list persons
+2.  FAKEJARVIS shows a list of persons
```

Comment:
It would be better if it is standardised across the UG the product name: FAKEJARVIS or F.A.K.E.J.A.R.V.I.S.